lib/foreman_remote_execution/engine.rb

```diff
@@ -56,6 +56,7 @@ class Engine < ::Rails::Engine
                                                 'api/v2/job_invocations' => [:create] }, :resource_type => 'JobInvocation'
           permission :view_job_invocations, { :job_invocations => [:index, :show, :auto_complete_search], :template_invocations => [:show],
                                               'api/v2/job_invocations' => [:index, :show, :output] }, :resource_type => 'JobInvocation'
+          permission :cancel_job_invocations, { 'api/v2/job_invocations' => [:cancel] }, :resource_type => 'JobInvocation'
```

Comment:
You're adding `:cancel_job_invocation` here, while adding `:edit_job_invocations` permission later to the `MANAGER_PERMISSIONS`.  I would probably stick with `:edit_job_invocations` here

Reply:
There is a double-save happening when a job invocation is being created by the composer (probably due to associations). This means a user has to have both `:create` and `:edit` permissions to be able to run jobs. I would like to go with `:cancel` before we get this sorted out

Reply:
I see, we should then unify the way how the UI and API uses the permission, by using `:cancel_job_invocations` for the UI as well